bloom-instance/service/backend/service.tf

```diff
@@ -5,6 +5,8 @@ module "service" {
   alb_map    = var.alb_map
   subnet_map = var.subnet_map
   dns        = var.dns
+  # CloudFront is unsupported for the backend service
+  cert_map = { "foo" : "bar" }
```

Comment:
is "foo" and "bar" just placeholders?

Reply:
I think they are there because cert_map variable is required but there is no cloudfront on the backend. This is going to need to be part of the eventual refactor.